src/js/contexts/AnnounceContext/stories/Polite.js

```diff
@@ -1,47 +1,67 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useContext, useState } from 'react';
+
+import { Announce } from 'grommet-icons';
 import { grommet } from 'grommet/themes';
-import { AnnounceContext, Box, Grommet, Heading, Text } from 'grommet';
+import {
+  Anchor,
+  AnnounceContext,
+  Box,
+  Button,
+  Grommet,
+  Heading,
+  Paragraph,
+  Text,
+} from 'grommet';
 
-const Announcer = ({ announce, message, mode, role }) => {
-  React.useEffect(() => {
-    const timeout = 3000;
-    announce(message, mode, timeout);
-  }, [announce, message, mode]);
+const message = `Thank you for clicking the Announce Button, 
+this announcement is currently being broadcast.`;
 
+const Announcer = ({ mode, role }) => {
+  const announce = useContext(AnnounceContext);
+  const [show, setShow] = useState(false);
   return (
-    <Text align="center" role={role} aria-live={mode}>
-      {message}
-    </Text>
-  );
-};
-
-Announcer.propTypes = {
-  announce: PropTypes.func.isRequired,
-  message: PropTypes.string,
-  mode: PropTypes.string,
-  role: PropTypes.string,
-};
+    <Box align="center" gap="medium">
+      <Button
+        label="Announce"
+        icon={<Announce />}
+        a11yTitle="Announce button"
+        reverse
+        onClick={() => {
+          setShow(true);
+          announce(message, mode);
+        }}
+      />
 
-Announcer.defaultProps = {
-  message: 'Here is a simple announcement. This will soon disappear',
-  mode: 'polite',
-  role: 'log',
+      {show && (
+        <Text role={role} aria-live={mode}>
+          {message}
```

Comment:
I don't think we should show the same message we are announcing. Screen readers will already do that via `aria-live`. I think we can remove this Text.